context/app/static/js/components/detailPage/visualization/Visualization/Visualization.tsx

```diff
@@ -118,7 +118,7 @@ function Visualization({
           leftText={shouldDisplayHeader ? <StyledSectionHeader>Visualization</StyledSectionHeader> : undefined}
           buttons={
             <Stack direction="row" spacing={1}>
-              {hasNotebook && <VisualizationWorkspaceButton uuid={uuid} />}
+              {hasNotebook && <VisualizationWorkspaceButton />}
```

Comment:
That's correct - this is specifically for the Visualization section though, which is only present for processed datasets.

Reply:
I think this will break the approach used in #3570 to make image pyramid notebooks work, though - is there a way we could keep that logic intact? I.e making it so that support image pyramid datasets use the parent dataset's ID to request a notebook.

Reply:
It looks like the update in https://github.com/hubmapconsortium/portal-ui/pull/3570 includes an additional parentUuid prop for the notebook download, not the notebook workspace - is the worry that the image pyramid workspaces will be impacted in some way?

Reply:
Never mind, I wasn't looking closely enough 😅 Got my wires crossed since the notebook endpoint includes `.ws`

Reply:
Ah no worries!